packages/@okta/vuepress-site/docs/reference/saml-hook/index.md

```diff
@@ -346,6 +348,10 @@ Okta supports URI claims with SAML assertion hooks. When you need to replace or
 
 In some scenarios, your service provider may require the `SessionNotOnOrAfter` attribute for the `<saml:AuthnStatement>` in the SAML assertion, which sets the provider session time correctly. Use `add` op with the path `/authentication/sessionLifetime` and a value for session lifetime in seconds to add this attribute. See the [Sample listing of JSON payload response](/docs/reference/saml-hook/#sample-listing-of-json-payload-of-response) for an example. Okta calculates `SessionNotOnOrAfter` by adding the `/authentication/sessionLifetime` value to the `issueInstant` attribute and returns it in the SAML assertion.
 
+## Timeout behavior
+
+After receiving the Okta request, if there is a response timeout or error, the Okta process flow proceeds with the original SAML assertion returned.
```

Comment:
```suggestion
If there is a response timeout or error after receiving the Okta request, the Okta process flow proceeds with the original SAML assertion returned.
```

Reply:
Removing "error" bit and recasting like other suggestions. Thanks.